torch/csrc/distributed/rpc/process_group_agent.cpp

```diff
@@ -254,12 +254,14 @@ void ProcessGroupAgent::sync() {
 }
 
 void ProcessGroupAgent::start() {
+  start_ = true;
```

Comment:
Doe this mean we can call start()/stop() multiple times on the same ProcessGroupAgent instance?

Reply:
Yes, and this requires clean shutdown. All statefule variables should be put in the correct state.